environments/dev/app/namespaces/overlays/ircc-cicd/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+
+namespace: ircc-cicd
+
+bases:
+- ../../../../../../components/app/namespaces/small
```

Comment:
for tekton, since they will be running a lot of jobs, I would give them more than a small project at least medium, if not large.